surveys/2022-annual-survey/questions.md

```diff
@@ -26,40 +26,62 @@ Type: select one
 
 ## For previous Rust users
 
-### As you have indicated that you're no longer using Rust, what prompted you to participate in this survey?
+### Why did you stop using Rust?
 
-Type: select all that apply (optional)
+Type: select all that apply
 
-- I plan to return to using Rust in the future
-- I consider myself part of the Rust community
-- Specifically to provide feedback on why I stopped using Rust
-- To provide feedback on Rust in general
-- Curiosity
-- Other (open response)
+- No benefit or not appropriate for my projects
+- Missing language features
+- Missing libraries
+- Missing tools
+- Too difficult to learn
+- Community was rude, unwelcoming, or otherwise off-putting
+- I prefer to use another language
+- Other
 
-> **justification**
->
-> Useful in understanding why non-users contribute;
+### Tell us more:
+
+Type: free text
+
+### Would you be likely to use Rust if the above issues were addressed
+
+Type: select one
+
+- Yes
+- No
 
 > **SURVEY FLOW**
 >
 > Skip to `## Your opinions about Rust` section
 
+TODO there are some questions before this which we could ask non-users, how do we make this work with the software?
```

Comment:
Which questions do you mean?

Reply:
Quite a lot of the survey, e.g., asking about Rust at work (just because they are a non-user doesn't mean their company doesn't use Rust)